app/mod_api/controllers.py

```diff
@@ -171,6 +171,57 @@ def get_metadata(publisher, package):
         return handle_error('GENERIC_ERROR', e.message, 500)
 
 
+@mod_api_blueprint.route("/dataproxy/<publisher>/<package>/<resource>.json", methods=["GET"])
```

Comment:
@pwalsh For now this is done for frontend javascript to handle with graphs and tables etc... 
We can continue discussions in #46 
Merging for now.